packages/components/src/Modal/ContextModal/ContextModal.module.scss

```
@@ -34,6 +34,10 @@
   color: $color-purple-800;
   text-align: left;
```

I think you can achieve the same affect by using `start` here, rather than implementing a `dir`

```suggestion
  text-align: start;
```

---

Okay, thanks for your feedback @gyfchong. I will make this change.